### PR TITLE
Migrate CSRF state to Redux Toolkit 

### PR DESCRIPTION
**Pull request by marialani**
## What are you doing in this PR?
This PR replaces the old Redux implementation for storing the CSRF state in the Redux store with a Redux Toolkit implementation. 

As the CSRF state is static on the frontend we've also removed a preexisting unused action [introduced in this PR](https://github.com/guardian/support-frontend/pull/126/files/6858df8665e58447c346db3a97e6bf0879ad14e3#diff-9c5c45bec6dd0e7253fcdbcfbdb99e0ff8be83131e5c5fa341e9e482528a5e27).

A lot of the file changes are down to updating imports or references to the csrf state, since it has been moved to the new `checkoutForm` reducer.

[**Trello Card**](https://trello.com/c/a59oFJWf/423-%F0%9F%9B%A0-redux-toolkit-migration-refactor-csr-state-%F0%9F%9B%A0)

Co-authored by: @GHaberis

### CODE REVIEW

**Comment by marialani** [comment]
[🛠 Redux Toolkit migration - refactor CSR state 🛠](https://trello.com/c/a59oFJWf/423-%F0%9F%9B%A0-redux-toolkit-migration-refactor-csr-state-%F0%9F%9B%A0)

**Review by i-hardy — APPROVED** [review]
This is great! I think in the future we could look at putting this inside another reducer, or taking it out of Redux entirely as it doesn't ever actually change, but it's really good to have it migrated.

File: support-frontend/assets/helpers/redux/checkout/csrf/reducer.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { initialCsrfState } from './state';
+
+export const csrfSlice = createSlice({
+	name: 'csrf',
+	initialState: initialCsrfState,
+	reducers: {},
+});
```
Comment (TomPretty) [inline support-frontend/assets/helpers/redux/checkout/csrf/reducer.ts:8]: So there are no reducers? i.e we never update this state?

Comment (GHaberis) [reply support-frontend/assets/helpers/redux/checkout/csrf/reducer.ts:8]: Yep, we found the unused action/reducer in the original PR that introduced it, and the comment in the PR notes it never gets used: https://github.com/guardian/support-frontend/pull/126/files#r127286535

**Review by TomPretty — APPROVED** [review]
Looks great! +1 to @i-hardy comment about maybe looking to remove this from redux in future if it's not being updated

**Pull request merged**